java/integTest/src/test/java/glide/TestUtilities.java

```diff
@@ -111,6 +112,17 @@ public static Map<String, String> parseInfoResponseToMap(String serverInfo) {
         return builder.useTLS(TLS);
     }
 
+    public static GlideClusterClientConfiguration.GlideClusterClientConfigurationBuilder<?, ?>
```

Comment:
lots of common code between GlideClientConfigurationBuilder, GlideClusterClientConfigurationBuilder and GlideClusterClientConfigurationBuilder